SPECS/kernel/kernel.spec

```diff
@@ -120,16 +129,15 @@ make mrproper
 
 %ifarch x86_64
 cp %{SOURCE1} .config
-arch="x86_64"
-archdir="x86"
 %endif
 
 %ifarch aarch64
 cp %{SOURCE2} .config
-arch="arm64"
-archdir="arm64"
 %endif
 
+arch=%{arch}
```

Comment:
Do we still need these two variables, if we're using RPM's macros now?

Reply:
I was worried about implicit uses of these environment variables inside `Makefile` or something similar, so I did not remove their definitions.

Reply:
I think in order for these to be used by `make` you'd either have to pass them through an argument like `make arch=...` or export them like `export arch=...`. Otherwise they should be local. Maybe there's also another way to pass your local env variables but I wouldn't expect that to be done by default.